fulfil_client/client.py

```diff
@@ -34,7 +34,7 @@ def wrapper(*args, **kwargs):
         if rv.status_code != requests.codes.ok:
             if 400 <= rv.status_code and rv.status_code < 500:
                 raise ClientError(loads(rv.text), rv.status_code)
-            raise ServerError(loads(rv.text), rv.status_code)
+            raise ServerError(rv.text, rv.status_code)
```

Comment:
we read the type and message from this a lot on importer app. This change will break all of that?